packages/components/src/logo/index.jsx

```diff
@@ -3,36 +3,41 @@ import PropTypes from 'prop-types';
 import clsx from 'clsx';
 import styles from './index.module.scss';
 
+import LogoText from '../logo_text';
+
+import { ReactComponent as GHLogo } from './images/logo.svg';
+import { ReactComponent as GHChristmasLogo } from './images/logo-christmas.svg';
 
 const Logo = (props) => {
-  const { compact, to, localeName, children, ...cleanProps } = props;
+  const { compact, christmas, to, localeName, children, ...cleanProps } = props;
   const className = clsx(styles.root, props.className);
+  const LogoImage = christmas ? GHChristmasLogo : GHLogo;
 
   let text;
   if (!compact) {
     text = (
       <span className={styles.wrap}>
-        <i className={`icon-logo_type_${localeName}`} />
+        <LogoText className={styles.text} localeName={localeName} />
         {children && <em>{children}</em>}
       </span>
     );
   }
 
-  const content = <><i className="icon-logo" />{text}</>;
+  const content = <><LogoImage className={styles.image} />{text}</>;
 
   if (to) return <a {...cleanProps} className={className} href={to}>{content}</a>;
-
   return <span {...cleanProps} className={className}>{content}</span>;
 };
 
 Logo.defaultProps = {
   compact: false,
-  localeName: 'de_de',
```

Comment:
that's a breaking change imo

Reply:
yes, but i checked and these logos are not in use anywhere, we have to upgrade manually anyway

Reply:
it's still a breaking change